### PR TITLE
Add README.md badges 

### PR DESCRIPTION
**Pull request by krupalitrivedi**
Solves issue #18 

The badges have been added for the repository with the help of  `shields.io`. 

Let me know if any changes are needed :)

### CODE REVIEW

**Comment by krupalitrivedi** [comment]
Hi :) you assigned this to me, are there any changes needed? if so let me know!

File: README.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ SPDX-License-Identifier: GPL-2.0-or-later
 
 # userstorage
 
+[![](https://img.shields.io/badge/pypi-v0.5.1-orange)](https://pypi.org/project/userstorage/)
```
Comment (nirs) [inline README.md]: This commit is fine in a private work repo, but not for a pull request.

File: README.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ SPDX-License-Identifier: GPL-2.0-or-later
 
 # userstorage
 
-[![](https://img.shields.io/badge/pypi-v0.5.1-orange)](https://pypi.org/project/userstorage/)
+[![](https://img.shields.io/badge/pypi-v0.5.1-orange)](https://pypi.org/project/userstorage/) [![](https://img.shields.io/badge/python-2.7%20%7C%203.6%20%7C%203.7%20%7C%203.8%20%7C%203.9%20%7C%203.10-blue)](https://pypi.org/project/userstorage/) [![](https://img.shields.io/badge/CI-passing-brightgreen)](https://github.com/nirs/userstorage/blob/master/.github/workflows/ci.yml)
```
Comment (nirs) [inline README.md]: Like previous commit, this is partial work in progress and should not be part of the pull request.

File: README.md [hunk 3]
```diff
@@ -5,7 +5,7 @@ SPDX-License-Identifier: GPL-2.0-or-later
 
 # userstorage
 
-[![](https://img.shields.io/badge/pypi-v0.5.1-orange)](https://pypi.org/project/userstorage/) [![](https://img.shields.io/badge/python-2.7%20%7C%203.6%20%7C%203.7%20%7C%203.8%20%7C%203.9%20%7C%203.10-blue)](https://pypi.org/project/userstorage/) [![](https://img.shields.io/badge/CI-passing-brightgreen)](https://github.com/nirs/userstorage/blob/master/.github/workflows/ci.yml)
+[![](https://img.shields.io/pypi/v/userstorage)](https://pypi.org/project/userstorage/) [![](https://img.shields.io/pypi/pyversions/userstorage)](https://pypi.org/project/userstorage/)  [![](https://img.shields.io/pypi/dm/userstorage)](https://pypi.org/project/userstorage/) [![](https://github.com/nirs/userstorage/actions/workflows/ci.yml/badge.svg)](https://github.com/nirs/userstorage/blob/master/.github/workflows/ci.yml) [![](https://img.shields.io/github/license/nirs/userstorage)](https://pypi.org/project/userstorage/)
```
Comment (nirs) [inline README.md]: The state of the file at this commit looks good!
https://github.com/nirs/userstorage/blob/698d3a326ee5aebb0f57f474d8f8e90612dc5346/README.md

However the source is little bit messy:
```
[![](https://img.shields.io/pypi/v/userstorage)](https://pypi.org/project/userstorage/) [![](https://img.shields.io/pypi/pyversions/userstorage)](https://pypi.org/project/userstorage/)  [![](https://img.shields.io/pypi/dm/userstorage)](https://pypi.org/project/userstorage/) [![](https://github.com/nirs/userstorage/actions/workflows/ci.yml/badge.svg)](https://github.com/nirs/userstorage/blob/master/.github/workflows/ci.yml) [![](https://img.shields.io/github/license/nirs/userstorage)](https://pypi.org/project/userstorage/)
```

It will be nicer if we put one badge per line - markdowns will render all badges in the same line:

```
[![](https://img.shields.io/pypi/v/userstorage)(https://pypi.org/project/userstorage/)
[![](https://img.shields.io/pypi/pyversions/userstorage)(https://pypi.org/project/userstorage/)
[![](https://img.shields.io/pypi/dm/userstorage)](https://pypi.org/project/userstorage/)
[![](https://github.com/nirs/userstorage/actions/workflows/ci.yml/badge.svg)](https://github.com/nirs/userstorage/blob/master/.github/workflows/ci.yml)
[![](https://img.shields.io/github/license/nirs/userstorage)](https://pypi.org/project/userstorage/)
```

Some lines are too long but markdown does not support line breaks where we would
like to have them.

Another issue is linking the CI badge ci.yaml file:
https://github.com/nirs/userstorage/blob/master/.github/workflows/ci.yml

A more useful link is the ci workflow:
https://github.com/nirs/userstorage/actions/workflows/ci.yml

To prepare a final pull request, please:
1. Squash all commits to single commit 
   https://stackabuse.com/git-squash-multiple-commits-in-to-one-commit/
2. Put one badge per line
3. Fix ci badge target
4. Force push to your repo to update the pull request

**Review by nirs — CHANGES_REQUESTED** [review]
Thanks @krupalitrivedi, the README is much nicer now!

**Comment by nirs** [comment]
@krupalitrivedi are you still working on this?

**Comment by krupalitrivedi** [comment]
@nirs Yes, I will update the PR in 1-2 days, got stuck in some school work.

File: README.md [hunk 4]
```diff
@@ -144,7 +154,7 @@ filesystem (/var/tmp/example-storage/file-4k-mount/file).
 ## Projects using userstorage
 
 - [ovirt-imageio](https://github.com/oVirt/ovirt-imageio)
-- [vdsm](https://github.com/oVirt/vdsm)
+- [vdsm](https://github.com/oVirt/vdsm) - embers a more recent version
```
Comment (nirs) [inline README.md]: This reverts #26, please remove this change.

File: README.md [hunk 5]
```diff
@@ -5,6 +5,16 @@ SPDX-License-Identifier: GPL-2.0-or-later
...
 
+[![](https://img.shields.io/pypi/v/userstorage)](https://pypi.org/project/userstorage/) 
+
+[![](https://img.shields.io/pypi/pyversions/userstorage)](https://pypi.org/project/userstorage/) 
+
+[![](https://img.shields.io/pypi/dm/userstorage)](https://pypi.org/project/userstorage/) 
+
+[![](https://github.com/nirs/userstorage/actions/workflows/ci.yml/badge.svg)](https://github.com/nirs/userstorage/actions/workflows/ci.yml) 
+
+[![](https://img.shields.io/github/license/nirs/userstorage)](https://pypi.org/project/userstorage/)
```
Comment (nirs) [inline README.md:12]: Looks good except the extra blank line between the badges, which renders one badge per
line, instead of all badges on the same line:
https://github.com/nirs/userstorage/blob/39623e381c3e8ae5cf7abe766498fd379ac92364/README.md

**Review by nirs — APPROVED** [review]
@krupalitrivedi thanks for your contribution!

I removed the accidental change the projects section and trailing whitespace from the badges lines.
